lib/convection/model/template.rb

```diff
@@ -11,6 +11,20 @@ module DSL
     module Template
       include DSL::Helpers
 
+      CF_MAX_BYTESIZE = 51200
```

Comment:
For numbers greater than 999 use the underscore character. `51_200` it helps legibility.